napari/components/viewer_model.py

```diff
@@ -588,13 +584,9 @@ def _on_add_layer(self, event):
         self._update_layers(layers=[layer])
 
         if len(self.layers) == 1:
+            # set dims slider to the middle of all dimensions
```

Comment:
Optional: I don't think you need the implementation comment and the well named method. I'd be slightly in favor of just the implementation comment given that `_go_to_center_step` is used once and is one line.

Reply:
I like the method on the dims, cause I figured it might be useful in other contexts, even though we're not currently using it. It might be nice to have a "center dims" exposed publicly.